### ISSUE TITLE
Installation fails on windows

### ISSUE THREAD

**Issue by DarkenLM** [Y2Kwastaken/CosmicTools#2]
Version: 2.0.2-SNAPSHOT
Command: `java -jar CosmicTools.jar --decompile --driver=CHROME --ignore-cache`

I found quite a few issues, so I'll detail them below:
1. Downloading seems to just not work, for some reason? With both drivers.
2. The only time it downloaded, it downloaded version `0.1.3` to the `v015` directory, causing the code to crash with NoSuchFileException.
3. After using it on a new directory, this error came out:
```
java.lang.RuntimeException: java.nio.file.NoSuchFileException: .\cosmic-reach\cosmic-reach-jar.zip
        at sh.miles.cosmictools.Utils.unzip(Utils.java:57)
        at sh.miles.cosmictools.stage.CosmicReachUnzipStage.runStage(CosmicReachUnzipStage.java:20)
        at sh.miles.cosmictools.CosmicTools.main(CosmicTools.java:55)
Caused by: java.nio.file.NoSuchFileException: .\cosmic-reach\cosmic-reach-jar.zip
        at java.base/sun.nio.fs.WindowsException.translateToIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source)
        at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(Unknown Source)
        at java.base/java.nio.file.Files.newByteChannel(Unknown Source)
        at java.base/java.nio.file.Files.newByteChannel(Unknown Source)
        at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(Unknown Source)
        at java.base/java.nio.file.Files.newInputStream(Unknown Source)
        at sh.miles.cosmictools.Utils.unzip(Utils.java:33)
        ... 2 more
```

**Comment by Y2Kwastaken** [comment]
Hey thanks for the issue, Are any of these issues able to be replicated on the previous version 2.0.1-SNAPSHOT? Proguard minimization may have removed classes that windows needs to download with Selenium. Given that this is a pretty likely scenario I will need you to find a ClassNotFoundException for me. If you can not create a stacktrace error after clearing all caching please let me know

**Comment by The-Epic** [comment]
After some testing, the chrome driver does not work at all for me, with or without --ignore-cache but firefox will work without --ignore-cache. It gives the same exception as OP's. All tests ran on 2.0.2-SNAPSHOT on windows with java 17 Temurin-17.0.9+9

**Comment by Y2Kwastaken** [comment]
Thank you for finding this issue <3 fixed with 2.0.3-SNAPSHOT
